hrs-portlets-ps-impl/src/main/resources/app-context/psAppContext.xml

```diff
@@ -71,6 +71,11 @@
                 <value>ROLE_VIEW_MANAGED_TIMES</value>
             </set>
         </entry>
+        <entry key="UW_DYN_PY_DIRDEP_SS">
+          <set>
+              <value>ROLE_VIEW_DIRECT_DEPOSIT</value>
```

Comment:
Named this MyUW representation of the role in the spirit of the existing role mappings: users with existing role `ROLE_VIEW_TIME_SHEET`, for instance, can *edit* their timesheets, not just view it. Likewise, users with this new role will be able to *edit* their direct deposit configuration, not just view it. "VIEW" in the role names seems to mean "view the link to launch the thing" rather than be about what one can do (view vs edit) on the thing.